src/components/Navbar/Navbar.css

```
@@ -1,59 +1,54 @@
-.dcl.navbar {
+.dui-navbar2 {
```

This change should be in all classNames and inside the component too
```suggestion
.dui-navbar {
```